### ISSUE TITLE
Create Message View

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#41]
## User story
1. As a user
2. I need a message view
3. So that I can see relevant information about messages

## Acceptance criteria
* [ ] Create the message view component that visualizes a list of messages according to the [mockup](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/blob/main/Documentation/amos_mockup.pdf) (for now these information should only be dummy data)
* [ ] Take the filters from the topic view and add a message filter to it
(consider that the content of the message filter should fulfill potentially previously selected filters)
* [ ] The message filter should provide a regex search

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
